### DISCUSSION TITLE
search “Discussions” from “Issues” or “Pull requests” tab

### DISCUSSION THREAD

**Discussion by locnide** [community/community#3792] [Discussions]
I often search for both issues and pull requests when I want to know the current state of a feature or a bug.
Now that there is a discussion tab where discussions about feature may take place, I think that it would be nice to be able to search across all three tabs : issues, pull requests and discussions.

Is it already possible ? Maybe users will try to do that with the `is:discussion` filter.

**Comment by hawkerm** [comment]
Yeah, I agree. I was just trying to find an old issue I was a part of. Couldn't find it at all. It had ended up being converted to a discussion. It's really hard to search across GitHub to find all places where I've been involved in a discussion whether it be a Discussion, Issue, or PR.

**Comment by brandonkelly** [comment]
Agree this would be good. It doesn’t really make any sense that it’s possible to search both issues and PRs by removing the `is:issue` or `is:pr` term from either of those sections, but that discussions wouldn’t be included.

FWIW, it’s possible to search across both issues and discussions from the global search bar, which defaults to searching the current repo:

<img width="614" alt="The global search bar, showing that results for the current repo will be returned by default" src="https://user-images.githubusercontent.com/47792/122971699-2cee8c80-d344-11eb-95c4-6db890bcef09.png">

And you can then find what you’re looking for by clicking on either **Issues** or **Discussions** in the sidebar:

<img width="809" alt="A search result page for a search within the craftcms/cms repo" src="https://user-images.githubusercontent.com/47792/122971954-78a13600-d344-11eb-9af4-802e38cd06d1.png">
